main.py

```diff
@@ -1,69 +1,4 @@
-class Node:
-    # This class provides a generic node for a linked list. 
-    def __init__(self, number, name):
-        self.value = int(number) #number element, in this case ssn
-        self.name = str(name) #name element attached to ssn
-        self.next = None #pointer
-
-class IRS_Single_Linked_List:
-    def __init__(self):
-        self.headPtr = None
-
-    def traverse_list(self):
-        if self.headPtr is None:
-            print("List has no element")
-            return
-        else:
-            iterative_node= self.headPtr
-            while iterative_node is not None:
-                print (iterative_node.value, " ", iterative_node.name, " ")
-                iterative_node = iterative_node.next
-
-    def insert_at_end(self, number, name):
-        new_node = Node(number, name)
-        if self.headPtr is None:
-            self.headPtr = new_node
-            return
-        iterative_node = self.headPtr
-        while iterative_node.next is not None:
-            iterative_node= iterative_node.next
-        iterative_node.next = new_node
-
-    # Search to see if an ssn exists in the SLL. If it does -- return True. Else return false 
-    def search_value(self, x):
-        if self.headPtr is None:
-            print("List has no elements")
-            
-        else: 
-            iterative_node = self.headPtr
-            while iterative_node is not None:
-                if iterative_node.value == x:
-                    print("value found")
-                    return True
-                iterative_node = iterative_node.next
-            print("value not found")
-        return False
-
-    def delete_element_by_value(self, x):
-        if self.headPtr is None:
-            print("The list has no element to delete")
-            return
-
-    # Deleting first node 
-        if self.headPtr.value == x:
-            self.headPtr = self.headPtr.next
-            return
-
-        iterative_node = self.headPtr
-        while iterative_node.next is not None:
-            if iterative_node.next.value == x:
-                break
-            iterative_node = iterative_node.next
-
-        if iterative_node.next is None:
-            print("value not found in the list")
-        else:
-            iterative_node.next = iterative_node.next.next
+from classes.singleLinkedList1 import LinkedList, Node
 
 ################# Driver Section ################## 
```

Comment:
All this code is github already. When you start this file, I recommend you start from scratch completly. This prevents you from clouding your thoughts by seeing old code.